.mailmap

```diff
@@ -18,7 +18,6 @@ Alex Griffing <argriffi@ncsu.edu> alex <argriffi@ncsu.edu>
 Alex Griffing <argriffi@ncsu.edu> argriffing <argriffi@ncsu.edu>
```

Comment:
Yep, looks like you are right. Entries of the form "A <B> <B>" are valid though, remaps possible nicknames based on address. Just found some decent .mailmap documentation at https://git-scm.com/docs/git-shortlog.